### DISCUSSION TITLE
BF207AD3-2641-2921-1B66-1A6D727F0855

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#131] [General]
# BF207AD3-2641-2921-1B66-1A6D727F0855

16 Psyche is the largest known M-type asteroid ever discovered, measuring 222 km in diameter. Located in the main asteroid belt, this metallic object is theorized to be an exposed core of a protoplanet. It will be studied up close by the NASA mission of the same name starting August 2029

https://celestia.mobi/addon?item=BF207AD3-2641-2921-1B66-1A6D727F0855&lang=zh_CN

**Comment by 13940951572** [comment]
为什么我下载这款小型星插件打开不显示小行星呢？

**Reply by DaveBowman2001** [reply]
SSC-code error caused the CMOD model to not appear. Should be fixed now with the new update (231110)
